src/main/java/net/snowflake/ingest/streaming/internal/ChunkMetadata.java

```diff
@@ -152,6 +162,11 @@ Integer getChunkLength() {
     return chunkLength;
   }
 
+  @JsonProperty("chunk_length_uncompressed")
```

Comment:
I believe this will require a server side change first

Reply:
server side change is merged

Reply:
but yes, we have to wait